pallets/asset/src/identifier.rs

```diff
@@ -0,0 +1,176 @@
+use codec::{Decode, Encode};
```

Comment:
Lets' move this stuff to `./primitives/src` since we have all similar stuff there.

Reply:
Okay that actually makes a lot more sense